hazelcast-enterprise/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12.3
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Shouldn't we have the specific version of the base image here? Otherwise, we risk the following issues:
- If RH pushes by mistake the wrong `latest` image, we have an issue
- Two same `docker build` can result in different results (if `latest` is changed in the meantime)
- Some of the commands below may stop working and some point and we discover it the hard way.

Reply:
Actually, their(RedHat Registries) latest tag usage is a little bit different from DockerHub, `latest` tag is like symbolic link at linux. They just change its pointer, do not push separate image for it. But your points are valid, I have changed it to specific version(8.3) 👍